terraform/parser.go

```diff
@@ -32,17 +32,10 @@ type NewParserParams struct {
 
 type JSON = map[string]interface{}
 
-func NewParserVariables(variables ModuleVariables) ValueMap {
```

Comment:
This function was used outside of the parser and it didn't make sense to be called `NewParserVariables`. It is now a utility function `createValueMap` although it's only used in two places. I'd consider removing it but I know this was added during a refactoring and I don't want to completely change the intention of that refactoring